### ISSUE TITLE
Heads up - This project is no longer maintained

### ISSUE THREAD

**Issue by mattlebel** [Crowdtilt/CrowdtiltOpen#267]
#259 

I'm one of the original developers on this project. If anyone comes across this in 2017 or beyond, please note that Crowdhoster / Crowdtilt Open is no longer under active development.

The corporate entity behind the project, Tilt.com, Inc. was acquired by Airbnb in early 2017. There are no plans to continue work on this.

While this project can no longer be spun up from the source code, feel free to ping me if you have any questions about self-hosted crowdfunding or pre-orders and I'll do my best to lend a hand: crowdhoster[at]lebel[dot]io.

**Comment by nightstryke** [comment]
So what you're saying is even if we download this it won't compile or run right?

**Comment by swyman** [comment]
That's correct. You could probably get it to boot, but all of the functionality you're actually interested in (campaigns, payments, user signup) all rely on the Tilt API, which is no longer running.

**Comment by nightstryke** [comment]
@swyman Well that's unfortunate.
